src/jekyll/_includes/shared/highlight.liquid

```diff
@@ -1,5 +1,9 @@
-  </div>
-</div>
+{% comment %}
+Embeds a image, description and link to a specific Udacity course on the page.
+TODO: Apply and clean up styles
+TODO: Verify all strings are localized
+{% endcomment %}
+
```

Comment:
No, this is the "base" template that's used by `remember.liquid`,  `takeaway.liquid`. I copied and pasted the comment and accidentally included the bit about it being a specific Udacity course. Will fix that bit.